dom5.ts

```diff
@@ -38,6 +38,15 @@ export function hasAttribute(element: Node, name: string): boolean {
   return getAttributeIndex(element, name) !== -1;
 }
 
+export function hasSpaceSeparatedAttributeValue(element: Node, name: string, value: string): boolean {
```

Comment:
It wasn't clear how functions are chosen for participation in the dom5 predicate API.  I was originally thinking this might best be abstracted as `hasAttrValueDelimited(name: string, value: string, delim: string)` too.  How would that be?